src/charts/LineChart/LineChart.tsx

```diff
@@ -62,8 +61,15 @@ export default function LineChart({
   // Look at the data structure and declare how to access the values we'll need.
   // ********************
 
+  // Null values must be removed from the dataset so as to not break our the
+  // Line generator function.
+  const cleanData = useMemo(
+    () => data.filter((el) => el[xKey] !== null && el[yKey] !== null),
+    [data]
+  );
+
```

Comment:
instead of copying this in each chart, we might want to add it to a util function and import it. @robcrock